juniper_official/Interfaces/check-lo0-address-ipv6-netconf.rule

```diff
@@ -0,0 +1,180 @@
+/*

```

Comment:
1. As we discussed move IPv4 and v6 loopback verifications into community.
2. Increase sensor and trigger frequency to 3 minutes

Reply:
updated the rule